docs/hyn/5.2/logging.md

```diff
@@ -11,7 +11,7 @@ Tenancy ships with a custom tenant aware logger which you can register in the lo
 - If no tenant is detected it will store the log file to:
 `storage/logs/`.
 
-Add a new `tenant` channel to the channels array in your `config/logger.php` like below:
+Add a new `tenant` channel to the channels array in your `config/logger.php` or rather `config/logging.php` for new Laravel version like below:
```

Comment:
This seems to be a typo. It should have read `logging.php`. Please modify that and remove you addition.

Great find!

Reply:
Thank you. Which other PRs do you mean?

[x] move the "Get current environment" to the identification.md instead.
[x] reduce the length of that section title, eg "retrieve current tenant" nothing more?
[] The TenancyFacade can be more easily used from the root namespace with \Tenancy, showing both options makes sense. --> any open todo here, which?
[x] Getting the current tenant should be done using the new tenant() method on the Environment; I know it overlaps with website() but it makes talking about the tenant as being the website easier. --> added both as a lot still base on old way and should be easy to find via google
[x] The last example makes less sense, you can retrieve the current website of the currently identified hostname using the tenant or website method on the environment, there should be no need to complicate this further? --> Removed